src/octoprint/plugins/backup/__init__.py

```diff
@@ -129,6 +123,18 @@ def get_template_configs(self):
             {"type": "wizard", "name": gettext("Restore Backup?")},
         ]
 
+    ##~~ Utility Methods
+
+    def build_backup_filename(self, backup_prefix=None):
+        if not backup_prefix:
+            if self._settings.global_get(["appearance", "name"]) == "":
+                backup_prefix = "octoprint"
+            else:
+                backup_prefix = self._settings.global_get(["appearance", "name"])
```

Comment:
Hm... I'm a bit worried about using the instance name here without any kind of sanitizing. What if there's a `/` in there or anything else? Should at least be run through basic sanitization or slugification.

Reply:
added slugify implementation (ripped from filemanager storage) and that seems to work properly now. Anything with slashes get converted to `-`.